src/spikeinterface/curation/curation_model.py

```diff
@@ -0,0 +1,360 @@
+from pydantic import BaseModel, Field, model_validator, field_validator
+from typing import List, Dict, Union, Optional, Literal, Tuple
+from itertools import chain, combinations
+import numpy as np
+
+
+class LabelDefinition(BaseModel):
+    name: str = Field(..., description="Name of the label")
+    label_options: List[str] = Field(..., description="List of possible label options", min_length=2)
```

Comment:
We we really need this `...` everywhere ?
I think this is clear that they are all mandatory expecet whe there is a default no ?
Is this common pydantic ?

Reply:
You mean the optional?